src/models/feature.h

```diff
@@ -52,9 +52,18 @@ class Feature : public QObject {
   // Returns true if it was enabled via DevMode
   bool isDevModeEnabled() const;
 
+  // Returns true if this is a newly introduced feature
+  bool isNew() const;
```

Comment:
Make these 2 methods inline:

```
bool isNew() const { return m_new; }
bool isMajor() const { return m_majorFeature; }
```

Reply:
Because you added the 2 methods, now you can change the Q_PROPERTY:

```
Q_PROPERTY(bool isNew READ  isNew CONSTANT)
```
the same for the major feature.